app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```diff
@@ -214,14 +214,16 @@ class SessionControlView(
                     super.onLayoutCompleted(state)
 
                     if (!context.settings().showHomeOnboardingDialog) {
-                        if (context.settings().shouldShowJumpBackInCFR) {
-                            JumpBackInCFRDialog(view).showIfNeeded()
-                        } else if (context.settings().showSyncCFR) {
```

Comment:
This `else if` was wrong because we don't attempt to show the sync CFR if `shouldShowJumpBackInCFR` is true. It is intended that we only show either the Jump Back In CFR or Synced Tab CFR and negate the other one when one of them is shown.